recipe/meta.yaml

```
@@ -37,42 +38,44 @@ outputs:
     script: build_mmcv.sh  # [not win]
     requirements:
       build:
-        - python                                   # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}       # [build_platform != target_platform]
-        - pytorch ={{ pytorch }}={{ proc_type }}*  # [build_platform != target_platform]
-        - numpy                                    # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}                   # [build_platform != target_platform]
+        - ninja
+        - numpy                                                # [build_platform != target_platform]
+        - python                                               # [build_platform != target_platform]
+        - pytorch                                              # [build_platform != target_platform]
+        - pytorch ={{ pytorch }}={{ torch_proc_type }}*        # [build_platform != target_platform]
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         # this adds matching cuda requirement to run deps using __cuda package
         - {{ compiler('cuda') }}    # [cuda_compiler_version != "None"]
-        - ninja
       host:
-        - python
+        - future
+        - numpy
         - pip
+        - pkg-config  # [unix]
+        - py-opencv
+        - python
+        - pytorch
+        - pytorch ={{ pytorch }}={{ torch_proc_type }}*
```

> One way to unlock this, though no guarantees, is to simply replace `={{ proc_type }}*` with `=*{{ proc_type }}*` everywhere (no other changes needed)
> 
> See comment [#16 (comment)](https://github.com/conda-forge/mmcv-recipe-feedstock/pull/16#issuecomment-1314002070)

Do you mean removing the `{{ pytorch }}` part like so to enable installing `mmcv-full` with either `pytorch::pytorch` or `conda-forge::pytorch`? LIke so?

```suggestion
        - pytorch =*={{ torch_proc_type }}*
```

---

Oops, posted at the same time, closing this thread.

---

Right, so it would be something like `1.12=cuda`? I still don't get why it's an equal sign and not a hyphen or dash...